drivers/nrf24l01/nrf24l01test.py

```diff
@@ -1,14 +1,40 @@
-"""Test for nrf24l01 module."""
-
-import struct
+"""Test for nrf24l01 module. Portable between MicroPython targets."""
+try:
+    import struct
+except ImportError:
+    import ustruct as struct
```

Comment:
Would it work with just `import ustruct as struct` for all boards?  That's a bit cleaner.